101-azurefirewall-create-with-zones/azuredeploy.json

```diff
@@ -0,0 +1,185 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "virtualNetworkName": {
+      "type": "string",
+      "metadata": {
+        "description": "Virtual network name"
+      }
+    },
+    "vnetAddressPrefix": {
+      "type": "string",
+      "defaultValue": "10.0.0.0/16",
+      "metadata": {
+        "description": "Virtual network address range"
+      }
+    },
+    "subnetName": {
+      "type": "string",
+      "defaultValue": "subnet1",
+      "metadata": {
+        "description": "Subnet Name"
+      }
+    },
+    "subnetPrefix": {
+      "type": "string",
+      "defaultValue": "10.0.0.0/24",
+      "metadata": {
+        "description": "Subnet prefix"
+      }
+    },
+    "azureFirewallSubnetAddressPrefix": {
+      "type": "string",
+      "defaultValue": "10.0.1.0/24",
+      "metadata": {
+        "description": "AzureFirewallSubnet prefix"
+      }
+    },
+    "firewallName": {
+      "type": "string",
+      "defaultValue": "firewall1",
+      "metadata": {
+        "description": "Azure Firewall name"
+      }
+    },
+    "publicIPName": {
+      "type": "string",
+      "defaultValue": "publicIP1",
+      "metadata": {
+        "description": "Public IP address name"
+      }
+    },
+    "location": {
+      "type": "string",
+      "defaultValue": "[resourceGroup().location]",
+      "metadata": {
+        "description": "Location for all resources. Only certain regions support zones."
+      }
+    },
+    "availabilityZones": {
+      "type": "array",
+      "defaultValue": [],
```

Comment:
since we're trying to showcase zones, add a non-null defaultValue

Reply:
Ok sure. Done